universal-application-tool-0.0.1/app/views/admin/programs/ProgramIndexView.java

```diff
@@ -185,6 +187,24 @@ Tag maybeRenderEditLink(
     }
   }
 
+  private Tag maybeRenderManageTranslationsLink(Optional<ProgramDefinition> draftProgram) {
+    if (draftProgram.isPresent()) {
+      String linkText = "Manage Translations →";
+      String linkDestination =
+          routes.AdminProgramTranslationsController.edit(
+                  draftProgram.get().id(), LocalizationUtils.DEFAULT_LOCALE.toLanguageTag())
+              .url();
+      return new LinkElement()
+          .setId("program-edit-link-" + draftProgram.get().id())
```

Comment:
Suggestion (blocking): would this conflict with program edit link? Would you use a different id?

Reply:
Ah great catch! Copy/paste error - I updated to `program-translations-link-%id%`